compiler/compile/OMRCompilation.cpp

```diff
@@ -2079,6 +2085,33 @@ void OMR::Compilation::verifyCFG(TR::ResolvedMethodSymbol *methodSymbol)
       }
    }
 
+void OMR::Compilation::verifyAndFixRdbarAnchors()
+   {
+   TR::NodeChecklist anchoredRdbarNodes(self());
+   for (TR::PreorderNodeIterator iter(self()->getStartTree(), self()); iter.currentTree() != NULL; ++iter)
+      {
+      TR::Node *node = iter.currentNode();
+      if (node->getOpCodeValue() == TR::treetop ||
+          node->getOpCode().isResolveOrNullCheck() ||
+          node->getOpCode().isStoreReg() ||
```

Comment:
Is being under a store reg really to be considered "safe" ? 
It's morally like being under a store to an auto which I don't think you consider safe.

Reply:
I thought reg store was not a real store but only an indicator for register allocation. The opcode doesn't return true for `isStore` either.

Reply:
It is not a real store but neither is to be considered an "anchor". i.e. usually a reg store is present where the original auto/parm store was and if it was illegal to have a readbar under a store for the first occurrence then it's equally illegal  for a readbar under a reg store to be the first occurrence.

Reply:
Thanks for the explanation. removed the store reg check.